### PR TITLE
Bug 1120013 - [Camera][Text Selection] Camera app to follow text selection pattern

### PR DESCRIPTION
**Pull request by cctuan**
https://bugzilla.mozilla.org/show_bug.cgi?id=1120013

### CODE REVIEW

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=dd3f883982cf)

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ffaa96e94bbf)

File: apps/camera/style/app.css [hunk 1]
```diff
@@ -19,7 +19,6 @@ html {
 
 body {
   background-color: #000;
-  -moz-user-select: none; /* 1 */
```
Comment (wilsonpage) [inline apps/camera/style/app.css]: I think this should still remain.

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=408818cdf870)

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8ab802a0e48e)

**Comment by try-server-hook** [comment]
[cctuan George (cctuan)](https://github.com/cctuan) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=62369dfb8516)

**Pull request merged**